diff -urN mailcap-2.1.9.orig/mime.types mailcap-2.1.9/mime.types
--- mailcap-2.1.9.orig/mime.types	Wed Oct 24 00:37:49 2001
+++ mailcap-2.1.9/mime.types	Fri May  9 11:19:56 2003
@@ -305,6 +305,7 @@
 application/x-koan		skp skd skt skm
 application/x-latex		latex
 application/x-netcdf		nc cdf
+application/x-ns-proxy-autoconfig	dat
 
 # This conflicts with audio/x-pn-realaudio-plugin, which is commented out below.
 application/x-rpm		rpm
