--- mailcap-2.1.31/mime.types	2010-08-26 19:40:09.069126869 +0300
+++ mailcap-2.1.31/mime.types	2012-01-22 21:57:54.126209709 +0200
@@ -481,7 +481,7 @@
 application/vnd.ms-ims				ims
 application/vnd.ms-lrm				lrm
 application/vnd.ms-playready.initiator+xml
-application/vnd.ms-powerpoint			ppt
+application/vnd.ms-powerpoint			ppt pps
 application/vnd.ms-project			mpp
 application/vnd.ms-tnef				tnef tnf
 application/vnd.ms-wmdrm.lic-chlg-req
@@ -843,6 +843,8 @@
 application/xop+xml				xop
 application/xslt+xml				xsl xslt
 application/xv+xml				mxml xhvml xvml xvm
+application/x-rar-compressed			rar
+application/x-windows-gadget			gadget
 application/zip					zip
 audio/32kadpcm					726
 # 3gp, 3gpp: video/3gpp
@@ -979,6 +981,7 @@
 audio/vnd.vmx.cvsd
 audio/vorbis
 audio/vorbis-config
+audio/webm					weba
 image/cgm
 image/fits					fits fit fts
 image/g3fax
@@ -1006,7 +1009,7 @@
 image/vnd.fujixerox.edmics-mmr			mmr
 image/vnd.fujixerox.edmics-rlc			rlc
 image/vnd.globalgraphics.pgb			pgb
-image/vnd.microsoft.icon			ico
+image/vnd.microsoft.icon
 image/vnd.mix
 image/vnd.ms-modi				mdi
 image/vnd.net-fpx
@@ -1017,6 +1020,7 @@
 image/vnd.svf
 image/vnd.wap.wbmp				wbmp
 image/vnd.xiff					xif
+image/x-icon					ico
 message/CPIM
 message/delivery-status
 message/disposition-notification
@@ -1257,6 +1261,7 @@
 application/x-troff-ms				ms
 application/x-ustar				ustar
 application/x-wais-source			src
+application/x-x509-ca-cert			crt
 application/x-xz				xz
 audio/midi					mid midi kar
 audio/x-aiff					aif aiff aifc
@@ -1286,6 +1291,7 @@
 text/x-pod					pod
 text/x-setext					etx
 text/x-vcard					vcf
+video/webm					webm
 video/x-flv					flv
 video/x-ms-asf					asx
 video/x-ms-wm					wm
