--- mailcap-2.1.38/mime.types~	2011-08-16 12:03:33.000000000 +0300
+++ mailcap-2.1.38/mime.types	2012-01-22 22:12:34.152935778 +0200
@@ -568,7 +568,7 @@
 application/vnd.ms-office.activeX+xml
 application/vnd.ms-officetheme			thmx
 application/vnd.ms-playready.initiator+xml
-application/vnd.ms-powerpoint			ppt
+application/vnd.ms-powerpoint			ppt pps
 application/vnd.ms-powerpoint.addin.macroEnabled.12	ppam
 application/vnd.ms-powerpoint.presentation.macroEnabled.12	pptm
 application/vnd.ms-powerpoint.slide.macroEnabled.12	sldm
@@ -990,6 +990,8 @@
 application/yang				yang
 application/yin+xml				yin
 application/zip					zip
+application/x-rar-compressed			rar
+application/x-windows-gadget			gadget
 audio/1d-interleaved-parityfec
 audio/32kadpcm					726
 # 3gp, 3gpp: video/3gpp
@@ -1133,6 +1135,7 @@
 audio/vnd.vmx.cvsd
 audio/vorbis
 audio/vorbis-config
+audio/webm					weba
 image/cgm
 image/fits					fits fit fts
 image/g3fax
@@ -1165,7 +1168,7 @@
 image/vnd.fujixerox.edmics-mmr			mmr
 image/vnd.fujixerox.edmics-rlc			rlc
 image/vnd.globalgraphics.pgb			pgb
-image/vnd.microsoft.icon			ico
+image/vnd.microsoft.icon
 image/vnd.mix
 image/vnd.ms-modi				mdi
 image/vnd.net-fpx
@@ -1176,6 +1179,7 @@
 image/vnd.svf
 image/vnd.wap.wbmp				wbmp
 image/vnd.xiff					xif
+image/x-icon					ico
 message/CPIM
 message/delivery-status
 message/disposition-notification
@@ -1420,6 +1424,7 @@
 application/x-troff-ms				ms
 application/x-ustar				ustar
 application/x-wais-source			src
+application/x-x509-ca-cert			crt
 application/x-xpinstall				xpi
 application/x-xspf+xml				xspf
 application/x-xz				xz
