--- mailcap-r2-1-54.orig/mime.types	2023-05-23 18:18:12.000000000 +0200
+++ mailcap-r2-1-54/mime.types	2026-03-14 01:16:13.146015595 +0100
@@ -2247,7 +2247,7 @@
 text/html-sandboxed				sandboxed
 text/x-pod					pod
 text/x-setext					etx
-video/webm					webm
+video/webm					webm weba
 video/x-annodex					axv
 video/x-flv					flv
 video/x-javafx					fxm
@@ -2262,3 +2262,7 @@
 video/x-sgi-movie				movie
 x-conference/x-cooltalk				ice
 x-epoc/x-sisx-app				sisx
+
+# PLD additions
+application/x-windows-gadget			gadget
+text/x-component				htc
