--- mailcap-2.1.31/mime.types	2010-08-26 19:28:52.629404880 +0300
+++ mailcap-2.1.31/mime.types	2010-08-26 19:32:31.572508121 +0300
@@ -843,6 +843,7 @@
 application/xop+xml				xop
 application/xslt+xml				xsl xslt
 application/xv+xml				mxml xhvml xvml xvm
+application/x-windows-gadget	gadget
 application/zip					zip
 audio/32kadpcm					726
 # 3gp, 3gpp: video/3gpp
@@ -1006,7 +1007,7 @@
 image/vnd.fujixerox.edmics-mmr			mmr
 image/vnd.fujixerox.edmics-rlc			rlc
 image/vnd.globalgraphics.pgb			pgb
-image/vnd.microsoft.icon			ico
+image/vnd.microsoft.icon
 image/vnd.mix
 image/vnd.ms-modi				mdi
 image/vnd.net-fpx
@@ -1017,6 +1018,7 @@
 image/vnd.svf
 image/vnd.wap.wbmp				wbmp
 image/vnd.xiff					xif
+image/x-icon					ico
 message/CPIM
 message/delivery-status
 message/disposition-notification
