--- run-mailcap	2018-04-02 09:44:40.000000000 +0200
+++ run-mailcap	2018-04-02 09:51:19.000000000 +0200
@@ -141,9 +141,10 @@
 #   $tmpfile = POSIX::tmpnam($name);
 #   unlink($tmpfile);
 
-    $cmd  = "tempfile --mode=600";
-    $cmd .= " --prefix $head" if $head;
-    $cmd .= " --suffix $tail" if $tail;
+    $cmd  = "mktemp -t ";	# -t is depreciated, but --tmpdir would not be handled by older mktemp
+    $cmd .= "$head" if $head;
+    $cmd .= ".$tail" if $tail;
+    $cmd .= "XXXXXX";
 
     $tmpfile = `$cmd`;
     chomp($tmpfile);
